index.js

```diff
@@ -13,7 +15,8 @@ const replacer = (match, group) => {
 };
 
 class WhiteOut extends Stream.Transform {
-  constructor (rules, options) {
+  constructor (rules, root, options) {
```

Comment:
Seems strange to keep adding arguments to the constructor when there is already an `options` object.